lib/devise_saml_authenticatable/strategy.rb

```diff
@@ -6,15 +6,14 @@ class SamlAuthenticatable < Authenticatable
       include DeviseSamlAuthenticatable::SamlConfig
       def valid?
         if params[:SAMLResponse]
-          response = OneLogin::RubySaml::Logoutresponse.new(params[:SAMLResponse], saml_config)
-          !(response.response.include? 'LogoutResponse')
+          OneLogin::RubySaml::Response.new(params[:SAMLResponse])
```

Comment:
It turned out that warden only uses the `authenticate!` and optional `valid?` method in the context of authentication (https://github.com/hassox/warden/wiki/Strategies), so we were able to remove the logout-specific behavior here.